spec/requests/item_presenter_spec.rb

```diff
@@ -26,20 +26,20 @@
 
   it "converts camelCase params to snake_case" do
     item_presenter = Api::V1::ItemPresenter
-    params = {name: "FooBar", sellIn: 5, quality: 11}
+    params = {name: "Foo", sellIn: 5, quality: 11, description: "Bar"}
 
     actual = item_presenter.snakecase_keys(params)
-    expected = {"name" => "FooBar", "quality" => 11, "sell_in" => 5}
+    expected = {"name" => "Foo", "sell_in" => 5, "quality" => 11, "description" => "Bar"}
```

Comment:
To be consistent, can you change the syntax to be like the rest, example line 29.

Reply:
change made... also required changing snakecase_keys(params) in item_presenter.rb to match new syntax